core/storage/email/gae_models.py

```diff
@@ -452,9 +298,6 @@ class GeneralFeedbackEmailReplyToIdModel(base_models.BaseModel):
     """
     # The reply-to ID that is used in the reply-to email address.
     reply_to_id = ndb.StringProperty(indexed=True, required=True)
-    # TODO (nithesh): Overriding last_updated of base model for migrating
```

Comment:
Hm. So what happens to models that were created between when this change was introduced, and the upcoming deployment?

Reply:
Hmmm... It looks like these newly created instances have last_updated = None. (These are instances created post migration, with the flag enabled). The last_updated field for the threads are working fine. Only the message model has the last_updated set as None. On switching to this PRs branch and adding on messages the last_updated field is set correctly for the new messages. 

However in the UI, the rendering is correct. The messages are ordered by their ID (which is sequential), and the time rendered beside the message is the created_on time. 

To rectify the lost data, I think we can populate the message's last_updated by its created_on time (There is no way to update a sent message right now, so we can assume that these two times should match).

Reply:
OK. So what's your deployment plan for this release?